### ISSUE TITLE
assests folder Removed from base dir

### ISSUE THREAD

**Issue by esube** [brollb/deepforge#191]
For some reason part of the app is expecting some of the UI icons for the network visualization under localhost:8080/assets/... etc directory and when it couldn't find them there, first it throws errors on the browser console. But, then somehow it grabs them elsewhere and you could see the network. 

But, moments before they are displayed correctly, you'd see the network with no icons for the layers.
Then, it flashes and replaces them with the actual icons. When you look at the console, you'd see the 404 errors (see the attached image).
![assets_404_errors](https://cloud.githubusercontent.com/assets/7385894/12273525/3dc06720-b934-11e5-8f43-102326eab0de.png)

**Comment by brollb** [comment]
It looks like it is related to https://github.com/brollb/deepforge/issues/185. Have you made sure your project is up to date (`git pull`)?

**Comment by esube** [comment]
Yeah, it looks like it is related. But, you closed that issue yesterday and I got a clean copy this morning. I will still pull and try to see if that fixes it if you have modified it today. 
But, it looks like some where some code is still referencing that assets directory although it didn't break the regular working process.

**Comment by brollb** [comment]
Can you check if your `config/configuration.base.js` reflects the change in https://github.com/brollb/deepforge/commit/7b5732f964c6657840f611dc5b1071e5f8b31b0b? 

It should just be the removal of one line

**Comment by brollb** [comment]
I just found this problem - although the svgs weren't used, they are still loaded in the newer versions (post 1.4.0) of webgme. This has been fixed in branch 173 and will be merged with the data refactor.

I am bending the rules of the feature branches here (by fixing two issues on one branch) bc these issues both change the seed file and I pretty much can't fix merge conflicts as it is a binary.
